examples/06_Combining Models.ipynb

```diff
@@ -4,21 +4,33 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Combining Prognostic Models"
+    "# 6. Combining Prognostic Models"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This section demonstrates how prognostic models can be combined. There are two times in which this is useful: \n",
+    "This section demonstrates how prognostic models can be combined. There are two instances in which this is useful: \n",
     "\n",
-    "1. When combining multiple models of different inter-related systems into one system-of-system model (i.e., [Composite Models](https://nasa.github.io/progpy/api_ref/prog_models/CompositeModel.html)), or\n",
+    "1. Combining multiple models of different inter-related systems into one system-of-system model** (i.e., [Composite Models](https://nasa.github.io/progpy/api_ref/prog_models/CompositeModel.html)), or\n",
```

Comment:
What does the "**" add at the end of this line? (Possibly a typo?)

Reply:
Yup, that's a typo - good catch, Katy!